cadence/contracts/NFTCatalog.cdc

```diff
@@ -160,10 +160,22 @@ pub contract NFTCatalog {
         }
     }
 
+    /*
+        Don't use this function, as its deprecation is imminent and an increasing catalog size may result in your code reaching its gas limit. 
```

Comment:
I think we should change this to be more direct.

Something like 

`
--------------------
DEPRECATED
-------------------

Use getCatalogKeys and forEachCatalogKey instead if obtaining all elements from the catalog is essential. See issue for details: https://github.com/dapperlabs/nft-catalog/issues/138
`